### PR TITLE
1.0.2

### PR DESCRIPTION
**Pull request by Portals**
Solves all issues in milestone v1.0.2

### CODE REVIEW

**Comment by Portals** [comment]
I have published on npm: `@cthit/react-digit.components@1.0.2-alpha.0`

File: src/elements/digit-text-area/DigitTextArea.element.jsx [hunk 1]
```diff
@@ -4,6 +4,12 @@ import TextField from "@material-ui/core/TextField";
 import useLayoutMaterialUi from "../../styles/material-ui/use-layout-material-ui";
 import makeStyles from "@material-ui/styles/makeStyles";
 
+const useFullFlex = makeStyles({
+    root: {
+        flex: "1"
+    }
+});
+
```
Comment (Hoidi) [inline src/elements/digit-text-area/DigitTextArea.element.jsx:12]: What's the difference between this and what was before?

**Comment by Hoidi** [comment]
I'm guessing #399 is resolved in 079c575e95db4a7d040c4ae8759247283f886020. #405 tho?

File: src/elements/digit-text-area/DigitTextArea.element.jsx (hunk 1 above)
Comment (Portals) [reply src/elements/digit-text-area/DigitTextArea.element.jsx:12]: I was having a performance issue in `DigitTextArea` for Gamma, I'm guessing this was the issue with calling makeStyles every render.

**Comment by Portals** [comment]
#405 shouldn't be in v1.02, my bad!

Right now testing the new version in mat and gamma!

**Pull request merged**